project/scripts/sbt

```diff
@@ -3,20 +3,17 @@
 # Usage:
 # ./sbt <cmd>
 
-set -e
-
 CMD="$1"
 
 if [ -z "$CMD" ]; then
   echo "Error: missing sbt command"
   exit 1
 fi
 
-# get the ivy2 cache
-ln -s /var/cache/drone/ivy2 "$HOME/.ivy2"
-
 # run sbt with the supplied arg
 sbt -J-Xmx4096m \
     -J-XX:ReservedCodeCacheSize=512m \
     -J-XX:MaxMetaspaceSize=1024m \
-    -Ddotty.drone.mem=4096m "$CMD"
+    -Ddotty.drone.mem=4096m \
```

Comment:
So we still use that thing?

Reply:
From what I understood, `-J-Xmx4096m` is an option passed to the VM that run `sbt` and `Ddotty.drone.mem=4096m` will be passed to the forked VMs